packages/artemisrgb-bin/artemisrgb-bin.pacscript

```diff
@@ -0,0 +1,42 @@
+pkgname=artemisrgb-bin
+pkgver=1.2025.0819.0
+gives="artemisrgb"
+url="https://artemis-rgb.com/"
+pkgdesc="A universal RGB control software"
+maintainer=("Rodrigo Tavares <dev@rodrigotavar.es>")
+arch=("x86_64")
+provides=("artemisrgb")
+conflicts=("artemisrgb" "artemisrgb-git")
+license=("custom:PolyForm Noncommercial License 1.0.0")
+depends=("dotnet-runtime-8.0 | dotnet-runtime-7.0" "hicolor-icon-theme" "libc6" "bash" "zlib1g" "fontconfig" "libstdc++6")
+makedepends=("unzip")
+options=("staticlibs")
+source=("${pkgname}-${pkgver}.zip::https://updating.artemis-rgb.com/api/artifacts/3795832408")
+sha256sums=('424b0abdc63a60709cbc217393c80d1112aa6f97a4b0e6a797c2fd7d59e6a790')
+noextract=("${pkgname}-${pkgver}.zip")
```

Comment:
Why can't this be extracted?

Reply:
This was left over from a previous version where I had a .desktop file and an icon file as sources so didn't want to copy all of it to the pkgdir. I'll remove it, thanks.